src/app/components/progress-indicator/waterfall-progress-indicator/index.html

```diff
@@ -0,0 +1,325 @@
+            </div>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-9.png" />
+              <div class="caption caption-success sky-text-success">Do group related properties and let users edit them in isolation after they complete waterfall progress indicators.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Anatomy">
+        <sky-docs-anatomy imagePath="~/assets/img/guidelines/progress-indicator/waterfall-progress-anatomy.png">
+          <sky-docs-anatomy-item anatomyNumber="1">Step indicator</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="2">Step label</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="3">Completed step action</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="4">Active step action</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="5">Waterfall title</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item isOptional="true" anatomyNumber="6">Step details</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item isOptional="true" anatomyNumber="7">Start over action</sky-docs-anatomy-item>
+        </sky-docs-anatomy>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Options">
+        <h3>Waterfall title</h3>
+        <div class="sky-text-block">
+          <p>We recommend including a title to reflect the context of the overall task. The title does not change when users progress through the waterfall steps.</p>
+        </div>
+        
+        <h3>Step details</h3>
+        <div class="sky-text-block">
+          <p>You can provide additional details about completed steps, but limit details to single pieces of information. Don’t go overboard because users can reopen modals to review choices as necessary.</p>
+        </div>
+        
+        <h3>Start over action</h3>
+        <div class="sky-text-block">
+          <p>You can include a start over action to let users erase all saved data from the waterfall process. This returns user to the first step or to an earlier point in the workflow if decisions occured before the waterfall progress indicator.</p>
+        </div>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Behavior & states">
+        <div class="sky-text-block">
+          <p>Waterfall progress indicators guide users through one active step at a time. When users complete a step, the progress indicator updates its step indicator and step action to the completed state and updates the next step indicator and step action to the active state. Users can review and edit completed steps, but this does not undo the completed state.</p>
```

Comment:
If we want to show examples like Todd suggested in the other PR for this, I think we could use an image of side-by-side progress indicators where the current step advances from left to right to show the transition from one step to the next. But I don't know how useful this would be.

Reply:
Ported over to #625.